docs/CONTRIBUTING.md

```diff
@@ -53,6 +53,16 @@ These types of issue cover everything that involves adding new features to SORMA
 * **Possible Alternatives:** If possible, provide alternative solutions in case your proposed feature can not be implemented in the way you outlined it for any reason. This is generally *optional* but might be requested at a later point in time.
 * **Additional Information:** If there is anything else you want to add to your request, you can put it here. *Optional*.
 
+### Task
+
+Something to be done but not directly changes anything the product. Examples: Preparing coming up features/changes, change processes on Github etc.
+
+* **Problem Description:** Document what problem exists to be solved/changed.
+* **Proposed Change:** Describe in as much detail as possible how you would like to change something.
```

Comment:
Proposed text:

Describe in as much detail how the proposed solution should look like and what should be done in order to achieve it.